### ISSUE TITLE
Where should $ units appear on Operations screen?

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/number-line-operations#82]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/593.
I noticed on the Operations screen, the $ unit only appears in the Initial Net Worth box, the operation description label, and the operation description text that appear when making changes before adding the operation. This seems inconsistent, particularly with the Net Worth box.

**Visuals**
![Inkeddollarsignswhere_LI](https://user-images.githubusercontent.com/41024075/103826139-7ddb2480-5033-11eb-9c86-ed3af079fe9e.jpg)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Number Line: Operations‬
URL: https://phet-dev.colorado.edu/html/number-line-operations/1.0.0-dev.12/phet/number-line-operations_all_phet.html
Version: 1.0.0-dev.12 2020-12-23 23:50:10 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
Language: en-US
Window: 1280x658
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by amanda-phet** [comment]
Agreed. Thanks for bringing it up @KatieWoe . 

I'd like to run it by @kathy-phet and @ariel-phet at design meeting tomorrow, since they might have a good idea about where are the best spots for units (if not all of those spots).

**Comment by amanda-phet** [comment]
Places we want to include a dollar sign: 

- Title of accordion box: Net Worth ($) 
- Point labels
- Also want to try on the net worth value (when you click the evaluate/= button)

**Comment by jbphet** [comment]
I've added support for the 1st and 3rd items listed above.  I started on adding support for the 2nd item - point labels - but this will affect common code, and after looking at the number line a bit more, I'm a little suspicious that we won't end up keeping this.  So, I'd like to make sure we *really* want this before I start on it.

It seems to me that to make this work, we will need to substantially reduce the size of the font used for the point labels on the "Operations" screen, since otherwise they will start to overlap.  Here are some screen shots of cases where I think this would occur:

![image](https://user-images.githubusercontent.com/3082831/104354369-561e0d80-54c6-11eb-8b99-662dd2c449ae.png)

![image](https://user-images.githubusercontent.com/3082831/104354418-659d5680-54c6-11eb-97fa-9019dff3356c.png)

Actually, as I was writing this, I realized I could do a fast prototype by just adding the dollar sign all the time.  Here's what it looked like for one of the cases above:

![image](https://user-images.githubusercontent.com/3082831/104354760-efe5ba80-54c6-11eb-9061-e546aa7dd58e.png)

So, are we sure that this capability should be added and, if so, shall I just reduce the font size of the point labels to the point where they don't overlap?

Assigning to @amanda-phet for discussion.
